apps/backend/message/src/createMessage.ts

```diff
@@ -0,0 +1,47 @@
+import { Request, Response, NextFunction } from "express";
+import { IDatabase } from "pg-promise";
+
+export const useValidateCreateMessage = (
+  req: Request,
+  res: Response,
+  next: NextFunction,
+) => {
+  // const { user_id: sender_id, content, listing_id, receiver_id } = req.body;
+  const sender_id = "5";
```

Comment:
Make sure this route has `req: AuthenticatedRequest`
```suggestion
  const sender_id = req.user.userId
```